ansible/roles/common/defaults/main.yaml

```diff
@@ -6,3 +6,13 @@ bin_dir: /usr/bin
 # The directory used by Ansible to temporarily store
 # files on Ansible managed systems.
 ansible_temp_dir: /tmp/.ansible/files
+
+# The director used for the docker config
+docker_config_dir: "/etc/sysconfig"
+
+#Some default values that may get modified in etc/tasks/main.yml
+is_atomic: false
+is_coreos: false
+has_rpm: false
+has_firewalld: false
+has_iptables: false
```

Comment:
I see, the list is taken from ``main.yml``. Still, we should do something about the ``is_coreos`` and alike facts. What about to define ``is_fedora``, ``is_redhat``, ``is_centos`` and others just to provide what system we are on. If a task needs distro version and other, one can use ``ansible_distribution`` and a like facts as has been done so far.